presto-hive/src/test/java/com/facebook/presto/hive/TestOrcPageSourceMemoryTracking.java

```diff
@@ -338,10 +339,15 @@ public void testTableScanOperator()
             assertFalse(operator.isFinished());
             Page page = operator.getOutput();
             assertNotNull(page);
-            page.getBlock(1);
+            page.getBlock(0);
             if (memoryUsage == -1) {
                 memoryUsage = driverContext.getSystemMemoryUsage();
-                assertBetweenInclusive(memoryUsage, 460000L, 469999L);
+                if (SystemSessionProperties.isAriaScanEnabled(driverContext.getSession())) {
```

Comment:
static import

Reply:
@oerling Orri, could you make a change to statically import SystemSessionProperties.isAriaScanEnabled ?